# Robot name goes here
husky: # change the name
  # default controller
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Controller for husky wheels
  Front_R_lateral_controller: # change the name
    type: effort_controllers/JointPositionController 
    joint: Front_R_lateral_joint
    pid : {p: 800.0 , i: 25.0 , d: 5}

  Front_L_lateral_controller: # change the name
    type: effort_controllers/JointPositionController 
    joint: Front_L_lateral_joint
    pid : {p: 800.0 , i: 25.0 , d: 5}
  
  wheel_BR_controller: # change the name
    type: velocity_controllers/JointVelocityController 
    joint: wheel_RB_joint
    pid : {p: 50.0 , i: 0.0 , d: 0.0}

  wheel_BL_controller: # change the name
    type: velocity_controllers/JointVelocityController 
    joint: wheel_LB_joint
    pid : {p: 50.0 , i: 0.0 , d: 0.0}

  joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint2
    pid: {p: 1500.0, i: 15.0, d: 20.0, i_clamp_min: -100.0, i_clamp_max: 100.0}

  joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint3
    pid: {p: 1700.0, i: 20.0, d: 60.0, i_clamp_min: -100.0, i_clamp_max: 100.0}

  joint4_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint4
    pid: {p: 1500.0, i: 20, d: 50.0, i_clamp_min: -100.0, i_clamp_max: 100.0}